framework/jtreg/issue845/checker/NotInPackageChecker.java

```diff
@@ -0,0 +1,7 @@
+import org.checkerframework.common.basetype.BaseTypeChecker;
+
+/**
+ * Checker that is in the default package.  This tests for Issue 845
+ * https://github.com/typetools/checker-framework/issues/845
```

Comment:
Small comment: we usually have this comment about which issue it is in the first lines of the file.
For JTReg tests, I would suggest having the link with the test driver, Test.java in this case.